package.json

```diff
@@ -192,7 +192,7 @@
     "moment": "^2.29.2",
     "moment-timezone": "~0.5.40",
     "nwsapi": "^2.2.1",
-    "path-to-regexp": "~8.0.0",
+    "path-to-regexp": "~1.9.0",
```

Comment:
Was this a downgrade?

Reply:
@elsamaryv yeah we don't need this downgrade. ~8.0.0. should be fine already. Can you please close this pr